### PR TITLE
add --wait to restart command

### LINKED ISSUE #431: okteto restart should wait until the services are up and running
**Is your feature request related to a problem? Please describe.**
When I run `okteto restart`, I always have to run `kubectl get pod` a few times to check if the restarted services are ready to be consumed.

**Describe the solution you'd like**
`okteto restart --wait` to block until all the services have been restarted and are healthy.

### PR DESCRIPTION
**Pull request by rberrelleza**
Fixes #431

### CODE REVIEW

File: pkg/k8s/pods/pod.go [hunk 1]
```diff
@@ -178,5 +178,47 @@ func Restart(dev *model.Dev, c *kubernetes.Clientset) error {
...
-	return nil
+	if !wait {
+		return nil
+	}
+
+	return waitUntilRunning(dev.Namespace, fmt.Sprintf("%s=%s", OktetoDetachedDevLabel, dev.Name), c)
+}
+
+func waitUntilRunning(namespace, selector string, c *kubernetes.Clientset) error {
+	t := time.NewTicker(1 * time.Second)
```
Comment (pchico83) [inline pkg/k8s/pods/pod.go]: I would add logs telling we are waiting for pods to be ready and a final log telling everything is up and running. Maybe a log for each pod that reaches the running state too.

File: cmd/restart.go [hunk 2]
```diff
@@ -38,11 +39,12 @@ func Restart() *cobra.Command {
 
 	cmd.Flags().StringVarP(&devPath, "file", "f", defaultManifest, "path to the manifest file")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "namespace where the exec command is executed")
+	cmd.Flags().BoolVarP(&wait, "wait", "w", false, "wait until pods are ready")
```
Comment (pchico83) [inline cmd/restart.go]: What about doing this the default (and maybe the unique) option?

File: pkg/k8s/pods/pod.go (hunk 1 above)
Comment (rberrelleza) [reply pkg/k8s/pods/pod.go]: I'm not a big fan of showing too many logs. I like linux'  no news is good news approach. How about we just show a log saying 'waiting for pods to restart', and the success one? @rlamana what do you think?

File: cmd/restart.go (hunk 2 above)
Comment (rberrelleza) [reply cmd/restart.go]: Im 50/50 on this one. Do you know if anyone other than us is using this command?

File: pkg/k8s/pods/pod.go (hunk 1 above)
Comment (pchico83) [reply pkg/k8s/pods/pod.go]: sounds good

Comment (rlamana) [reply pkg/k8s/pods/pod.go]: I also think that way. "Waiting > Success" sounds good to me.

**Comment by rberrelleza** [comment]
Changed it to use a spinner like we do in up and down instead. Thoughts @pchico83 @rlamana ?

restart

**Comment by rlamana** [comment]
@rberrelleza looks really good! and it is consistent with the rest of outputs.

**Pull request merged**